CHANGELOG.md

```diff
@@ -10,6 +10,13 @@ Ansible Changes By Release
 * Removed the previouslly deprecated 'accelerate' mode and all associated keywords and code.
 * New simpler and more intuitive 'loop' keyword for task loops
 
+### Minor Changes
+* added a few new magic vars corresponding to configuration/command line options:
+  `ansible_diff_mode`, `ansible_inventory_sources`, `ansible_limit`, `ansible_run_tags` , `ansible_forks` and `ansible_skip_tags`
+* Updated the bundled copy of the six library to 1.11.0
+* Added support to `become` `NT AUTHORITY\System`, `NT AUTHORITY\LocalService`, and `NT AUTHORITY\NetworkService` on Windows hosts
+* New CLI options for ansible-inventory, ansible-console and ansible to allow specifying a playbook_dir to be used for relative search paths.
```

Comment:
Did you only intend to move some of these from the 2.4 section to the 2.5 section?  I know that the bundled six occurred in 2.4.0, for instance.

Reply:
I did not move those across 2.4 boundary, i just moved minor features above deprecations inside the 2.5 section

Reply:
Ah... we updated six in both 2.4 and 2.5.  Okay, this is fine, then.